ember-cli-build.js

```diff
@@ -5,6 +5,9 @@ const EmberAddon = require('ember-cli/lib/broccoli/ember-addon');
 module.exports = function (defaults) {
   const customBuildConfig = {
     // Add options here
+    '@appuniversum/ember-appuniversum': {
+      dutchDatePickerLocalization: true,
+    },
```

Comment:
This only affects the test app so consuming apps need to add that config as well. We _could_ do it magically, but I think it's better if the app defines that config.